index.html

```diff
@@ -315,51 +316,10 @@ <h2 class="title_contact">Contact me</h2>
 						</li>
 					</ul>
 				</form>
-				<div class="bar"></div>
+				<footer class="bar"></footer>
```

Comment:
- [ ] (optional) To make your code semantic wrap your form inside `<footer>` and place the footer outside of `<main>`